### PR TITLE
media-gfx/slic3r: bump to 1.3.0

### PR DESCRIPTION
**Pull request by l29ah**

### CODE REVIEW

**Comment by axs-gentoo** [comment]
I would recommend just setting S directly within each side of the PV==9999 conditional rather than setting the MY_WORKDIR temporary variable only to use that to set S.  Otherwise though, if it builds and runs fine, LGTM.

**Comment by l29ah** [comment]
It is not only used for $S.

**Comment by axs-gentoo** [comment]
That's true, but the place it's used in src_prepare() is also redundant, since src_prepare() operates in $S.  So that push and pop can be removed as well.

File: media-gfx/slic3r/slic3r-1.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils perl-module
```
Comment (Zlogene) [inline media-gfx/slic3r/slic3r-1.3.0.ebuild]: `make_desktop_entry` is currently provided by the `desctop.eclass`, so replace eutils

**Comment by l29ah** [comment]
It's nice to be a gentoo dev: no one asks you for irrelevant fixes when you revbump.

**Pull request closed without merging**